base/sysinfo.jl

```diff
@@ -589,4 +645,10 @@ function username()
     return pw.username
 end
 
+function show(io::IO, ::MIME"text/plain", cpu::AbstractVector{CPUinfo})
```

Comment:
Do we not consider this type piracy? Probably better to change something like this in a separate PR

Reply:
It's not type piracy, because it's specific to the `CPUinfo` type defined in this package.

I guess we could put it in another PR, but it makes no sense to me to document the `Sys.cpu_summary()` function in the manual when it is clearly just a pretty-printing method that pre-dates the 3-argument `show` machinery.  Seems clearer to just fix the `display` at the same time we document `Sys.cpu_info()`.

Reply:
So this bit of trivia is funny to me, since I don't really care either way, but this method was actually originally here, even before the 3-arg show, and it was way back in the day when we talked about the content of commits in person more often than on PRs (this commit was reviewed offline): https://github.com/JuliaLang/julia/commit/7dcdbd42bb868db6561592c9aa71688857f774bd

Reply:
Fair enough.  Maybe better to just remove this `show(io::IO, ::MIME"text/plain", cpu::AbstractVector{CPUinfo})` method (and references thereto) for now.

(Still seems weird to me that any of this stuff is in the public API, when I somehow doubt that was originally intended … seems like it would have been thought through more carefully, e.g. ditch the weird field names containing `!`.)

Reply:
so should I keep `show(io::IO, ::MIME"text/plain", cpu::AbstractVector{CPUinfo})` or remove it , or it will be handled in separate PR.

Reply:
Let's remove it, since it's controversial.